lib/tessel/wifi.js

```diff
@@ -185,11 +178,9 @@ Tessel.prototype.setWiFiState = function(enable) {
             }
           });
         } else {
-          this.connection.end();
```

Comment:
All connection ending should happen by consumers of this API. Otherwise things like the test rig can't run multiple commands on a device.